plugins/default/lib/component_health.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import nagios
```

Comment:
Everyone has their favorite formula for organizing imports in Python. As a matter of organization, I tend to follow a formula that matches the Python's style guide, a.k.a. PEP8:

```python
import stdlib1
import stdlib2

import externallib1
import externallib2

import internallib1
import internallib2
```

The imports are organized into groups, with an empty line separating them. The first group are things from the Python standard library, then external packages/modules (we don't have any), then local pkgs/mods (the code we wrote ourselves).

This makes it easier to identify a file's dependency with a quick glance at the first lines.

The pattern is documented in https://www.python.org/dev/peps/pep-0008/#imports

@mikenairn wdyt?